src/main/cpp/configs/RobotConfig.cpp

```diff
@@ -89,6 +89,10 @@ void RobotConfig::DefineMechanisms()
 {
 }
 
+void RobotConfig::DefineLEDs()
+{
+}
```

Comment:
@muscatd75 Should we be changing RobotConfig.cpp and .h? Isn't this generated

Reply:
Yes, this is the default function and can be overridden by the specific robots